src/io/ktor/experimental/client/redis/Redis.kt

```diff
@@ -88,9 +118,25 @@ class RedisClient(
     }
 
     override suspend fun execute(vararg args: Any?): Any? {
-        val result = CompletableDeferred<Any?>()
-        postmanService.send(RedisRequest(args, result))
-        return result.await()
+        return when (rmode) {
```

Comment:
Could you explain what is the reason for skipping `await` and why we can't use launch instead?

Reply:
Sending order is important. Would `launch` preserve the order and everything would work as expected? That could be an optimization if that's the case, but if it fails sometimes that would be a problem and a source of bugs.

Redis offers three modes for processing replies:
https://redis.io/commands/client-reply
```
ON. This is the default mode in which the server returns a reply to every command.
OFF. In this mode the server will not reply to client commands.
SKIP. This mode skips the reply of command immediately after it.
```

Image that you are reading from a file a lot of data and you want to insert it in Redis. You don't want to wait for replies. This allow bulk inserting.

This is tested here:
```kotlin
@Test
fun testDisableProcessing() = redisTest {
    RedisClient(address, maxConnections = 1, password = REDIS_PASSWORD).apply {
        clientReplyOff {
            del(key1)
            lpush(key1, "a", "b", "c")
        }
        assertEquals(listOf("a", "b", "c"), lgetall(key1).sorted())
    }
}
```